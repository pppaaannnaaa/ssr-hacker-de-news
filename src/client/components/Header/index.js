import React from 'react';

const Header = () => {
  return (
    <div className="navbar-fixed" role="navigation" aria-label="News de Hacker Logo">
      <nav className="blue">
        <div className="container">
          <div className="nav-wrapper">
            <a href="/" className="brand-logo" style={{ whiteSpace: 'nowrap' }}>
              News de Hacker
            </a>
          </div>
        </div>
      </nav>
    </div>
  );
};

export default Header;
