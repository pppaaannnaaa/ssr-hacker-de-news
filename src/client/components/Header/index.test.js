import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Header from './index';

describe('<Header />', () => {
  let props;

  beforeEach(() => {
    props = {};
  });

  afterEach(cleanup);

  test('Renders <Header /> component correctly', () => {
    const { container, getByText } = render(<Header />);
    expect(container).toMatchSnapshot();
    expect(getByText('News de Hacker')).toBeInTheDocument();
  });
});
