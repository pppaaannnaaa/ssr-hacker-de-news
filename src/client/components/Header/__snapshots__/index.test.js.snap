// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Renders <Header /> component correctly 1`] = `
<div>
  <div
    aria-label="News de Hacker Logo"
    class="navbar-fixed"
    role="navigation"
  >
    <nav
      class="blue"
    >
      <div
        class="container"
      >
        <div
          class="nav-wrapper"
        >
          <a
            class="brand-logo"
            href="/"
            style="white-space: nowrap;"
          >
            News de Hacker
          </a>
        </div>
      </div>
    </nav>
  </div>
</div>
`;
